content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -102,9 +102,12 @@ post](https://kubernetes.io/blog/2016/11/visualize-kubelet-performance-with-node
 explains how the dashboard can be interpreted to come up with a suitable
 `kube-reserved` reservation.
 
-In addition to `cpu`, `memory`, and `ephemeral-storage`, `pid` may be
-specified to reserve the specified number of process IDs for
-kubernetes system daemons.
+In addition to `cpu`, `memory`, and `ephemeral-storage`, `pid` and `hugepages-<size>` may also
+be reserved. `pid` may be specified to reserve a given number of process IDs for
+kubernetes system daemons. `hugepages-<size>` (read more about huge page resoureces
+[in this doc](/docs/tasks/manage-hugepages/scheduling-hugepages.md)) may be specified to
```

Comment:
```suggestion
in [Manage HugePages](/docs/tasks/manage-hugepages/scheduling-hugepages.md)) may be specified to
```
(also consider rewording this sentence to use the active voice).

Reply:
This paragraph overall seems to be getting unwieldy. Is there a different way to convey the same information?